getting-started.rst

```diff
@@ -17,7 +17,7 @@ In most cases, users of ODK:
 
 This requires:
```

Comment:
Maybe it would be helpful to suggest starting by trying out Collect with the default server and the all widgets form to get a sense of what's possible?